src/components/pages/maintenance/summary/summary.js

```diff
@@ -36,54 +31,64 @@ export const Summary = ({
   timeInterval,
   ...props
 }) => [
-  <ContextMenu key="context-menu">
-    <DeviceGroupDropdown />
-    <RefreshBar
-      refresh={props.refreshData}
-      time={props.lastUpdated}
-      isPending={alertProps.isPending || jobProps.isPending}
-      t={props.t} />
-    <TimeIntervalDropdown
-      onChange={onTimeIntervalChange}
-      value={timeInterval}
-      t={props.t} />
-    <ManageDeviceGroupsBtn />
-  </ContextMenu>,
-  <PageContent className="maintenance-container summary-container" key="page-content">
-    <h1 className="maintenance-header">{props.t('maintenance.title')}</h1>
-    <div className="stat-container">
-      <div className="stat-group">
-        <StatCell value={renderUndefined(alertCount)} label={props.t('maintenance.openAlerts')} />
-        <div className="stat-column">
-          <StatCell
-            className="critical"
+    <ContextMenu key="context-menu">
+      <DeviceGroupDropdown />
+      <RefreshBar
+        refresh={props.refreshData}
+        time={props.lastUpdated}
+        isPending={alertProps.isPending || jobProps.isPending}
+        t={props.t} />
+      <TimeIntervalDropdown
+        onChange={onTimeIntervalChange}
+        value={timeInterval}
+        t={props.t} />
+      <ManageDeviceGroupsBtn />
+    </ContextMenu>,
+    <PageContent className="maintenance-container summary-container" key="page-content">
+      <PageTitle titleValue={props.t('maintenance.title')} />
+      <StatContainer>
+        <StatCell>
+          <StatProperty
+            value={renderUndefined(alertCount)}
+            label={props.t('maintenance.openAlerts')}
+            isLarge={true} />
+        </StatCell>
+        <StatCell>
+          <StatProperty
             value={renderUndefined(criticalAlertCount)}
             label={props.t('maintenance.critical')}
-            svg={svgs.critical} />
-          <StatCell
-            className="warning"
+            svg={svgs.critical}
+            svgClassName="statProp-critical" />
+          <StatProperty
             value={renderUndefined(warningAlertCount)}
             label={props.t('maintenance.warning')}
-            svg={svgs.warning} />
-        </div>
+            svg={svgs.warning}
+            svgClassName="statProp-warning" />
+        </StatCell>
+        <StatCell>
+          <StatProperty
+            value={renderUndefined(failedJobsCount)}
+            label={props.t('maintenance.failedJobs')}
+            isLarge={true} />
```

Comment:
Use size instead of isLarge to match what is used in the Indicator component.

Reply:
done